goaviatrix/transit_gateway_peering.go

```diff
@@ -132,6 +138,10 @@ func (c *Client) GetTransitGatewayPeeringDetails(transitGatewayPeering *TransitG
 	transitGatewayPeering.PrependAsPath1 = data.Results.Site1.ConnBGPPrependAsPath
 	transitGatewayPeering.PrependAsPath2 = data.Results.Site2.ConnBGPPrependAsPath
 
+	if len(data.Results.Tunnels[0].LicenseId) == 1 {
```

Comment:
Does `data.Results.Tunnels` always have at least 1 element in the slice? This would panic if the slice was empty

Reply:
There's at least one license assigned if two gateways are connected.